runtime/compiler/control/rossa.cpp

```diff
@@ -156,7 +156,7 @@ bool isQuickstart = false;
 TR::Monitor *vpMonitor = 0;
 
 
-char *compilationErrorNames[]={
+const char *compilationErrorNames[]={
```

Comment:
It looks like this change to `compilationErrorNames` that slipped in in the most recent commit is causing link errors on Windows builds.  It's still declared as `char * []` in [CompilationThread.cpp](https://github.com/eclipse-openj9/openj9/blob/a3acdc018def07c78c10ac70b60401d413b01e3e/runtime/compiler/control/CompilationThread.cpp#L989).

```
23:39:57  CompilationThread.cpp.obj : error LNK2001: unresolved external symbol "char * * compilationErrorNames" (?compilationErrorNames@@3PAPEADA)
23:39:57    Hint on symbols that are defined and could potentially match:
23:39:57      "char const * * compilationErrorNames" (?compilationErrorNames@@3PAPEBDA)
23:39:57  ..\j9jit29.dll : fatal error LNK1120: 1 unresolved externals
```

Reply:
Sneaky... done!

Reply:
As it turns out, this change causes a different failure in [CompilationThread.cpp](https://github.com/eclipse-openj9/openj9/blob/72c736633734a95902bd6b18fee46683518df641/runtime/compiler/control/CompilationThread.cpp#L1267) which would require a change in [omr/infra/Statistics.hpp](https://github.com/eclipse/omr/blob/25c4977ddc80f8f2b8425989b6ccc399c9d0b34c/compiler/infra/Statistics.hpp#L281). I could open a new PR for that and we could do a coordinated merge, but since this and related changes occur in `control` we could move them to step 3 which needs a coordinated merge anyway. I figure that would be easiest.